integration_tests/src/main/python/conditionals_test.py

```diff
@@ -137,3 +137,12 @@ def test_ifnull(data_gen):
                 'ifnull({}, b)'.format(s1),
                 'ifnull({}, b)'.format(null_lit),
                 'ifnull(a, {})'.format(null_lit)))
+
+
+@pytest.mark.parametrize('data_gen', single_level_array_gens_no_decimal, ids=idfn)
```

Comment:
nit: Could you add in a comment about why this test is not combined with `test_case_when` above and what is needed to combine them, preferably pointing to the github issue that would complete that work?

Reply:
Thanks for review, added